addons/web/static/src/views/no_content_helpers.xml

```diff
@@ -5,6 +5,12 @@
         <div class="o_view_nocontent">
             <div class="o_nocontent_help">
                 <t t-out="noContentHelp"/>
+                <t t-set="facets" t-value="this.env.searchModel.facets"/>
+                <t t-if="facets and !(facets.length===0)">
+                    <button 
+                        class="btn text-nowrap me-1 btn-primary my-2 o_reset_filter_button"
+                        t-on-click="this.removeFilter">Reset Filters</button>
```

Comment:
I think clearQuery on search model should be used and a check on query.length. This would remove all the methods in the controllers/renderers (method defined twice for graph?)